lib/slack/web/api/endpoints/chat.rb

```diff
@@ -287,7 +287,7 @@ def chat_unfurl(options = {})
           # @see https://github.com/slack-ruby/slack-api-ref/blob/master/methods/chat/chat.update.json
           def chat_update(options = {})
             throw ArgumentError.new('Required arguments :channel missing') if options[:channel].nil?
-            throw ArgumentError.new('Required arguments :text, :attachments or :blocks missing') if options[:text].nil? && options[:attachments].nil? && options[:blocks].nil?
+            throw ArgumentError.new('Required arguments :text, :attachments, :blocks or :reply_broadcast missing') if options[:text].nil? && options[:attachments].nil? && options[:blocks].nil? && options[:reply_broadcast].nil?
```

Comment:
Yes.
So I will have to also update chat.1.patch file after next API update, I think.

I wait for  next API update, and update this PR.

Reply:
Yes, but don't wait. Do the update.

Reply:
I see.
I also updated chat.1.path.
local execution of `bundle exec rake slack:api:update` did not overwrite this file.

Reply:
The slack-api-ref should have been updated first. I ran `rake api:update` in slack-api-ref and committed https://github.com/slack-ruby/slack-api-ref/commit/b38a62eacfb9bceb6ee01becadcd210b6b04c7d3. This should have been done by a cron, but that's dead because Travis-CI is no longer free, there's https://github.com/slack-ruby/slack-api-ref/issues/45 open to fix.

I ran `rake api:update` in this project and PRed https://github.com/slack-ruby/slack-ruby-client/pull/405.

Now you can rebase this change.